src/liquid/components/ld-select/ld-select.tsx

```diff
@@ -248,6 +255,7 @@ export class LdSelect implements InnerFocusable {
           )
           moreItem.innerText = `+${overflowingTotal}`
           if (moreItem.offsetTop < maxOffset) {
+            /* istanbul ignore next */
```

Comment:
1. To indicate that I'm not covering this line with any test on purpose.
2. To clean up the coverage report of anything that I don't want to cover, in order to see better what else needs test coverage.
3. Branch coverage is only shown in the report as soon as statement coverage reaches 100%.

Reply:
Okay, just make sure you're not tricking yourself here by aiming at 100% coverage. It may be legit to explicitly know that not all code is covered and  it might be possible (not always worth it) to add unit tests for those cases, anyway.

Reply:
I know I know 😇